src/copa_game/routes.clj

```diff
@@ -15,7 +16,7 @@
 
    ["/read"
     {:get {:summary "get all teams"
-           :responses {200 {:body [s/Any]}
+           :responses {200 {:body [out/Team]}
                        500 {:body s/Str}}
            :handler ports.http-in/get-teams}}]
```

Comment:
I see no tests for this. Should we start adding them or is that something you'd like to do later?

Reply:
Add integration test.